sugoi-api-ldap-utils/src/main/java/fr/insee/sugoi/ldap/utils/mapper/RealmLdapMapper.java

```diff
@@ -54,6 +54,9 @@ public static Realm mapFromSearchEntry(SearchResultEntry searchResultEntry) {
           realm.addProperty(GlobalKeysConfig.APP_MANAGED_ATTRIBUTE_PATTERNS_LIST, property[1]);
         } else if (property[0].equalsIgnoreCase("sort_key")) {
           realm.addProperty(LdapConfigKeys.SORT_KEY, property[1]);
+
+        } else if (property[0].equalsIgnoreCase("description")) {
```

Comment:
Why not use the description attribute directly ?

Reply:
Because i ddin't think to this ^^ i've done the change